components/Storyblok/SbHomepageMvp.tsx

```diff
@@ -5,15 +5,12 @@ import { HomepageSplitHero } from '../Homepage/HomepageSplitHero';
 import { Masthead } from '../Masthead';
 import { Changemaker } from '../Homepage/Changemaker';
 import { IdealFellow } from '../Homepage/IdealFellow';
-import { getProcessedImage } from '@/utilities/getProcessedImage';
-import { type SbImageType } from './Storyblok.types';
 
 type SbHomepageMvpProps = {
   blok: {
     _uid: string;
     title?: string;
     intro: string;
-    featuredStories?: SbBlokData[];
```

Comment:
Remove the "featuredStories" region in Homepage MVP page type in Storyblok, because they're mixing the featured stories with other content now. So basically everything gets absorbed into the `content` region